test/fixtures/wpt/WebCryptoAPI/getRandomValues.any.js

```diff
@@ -13,7 +13,17 @@ test(function() {
     assert_throws_dom("TypeMismatchError", function() {
         self.crypto.getRandomValues(new Float64Array(65537))
     }, "Float64Array (too long)")
-}, "Float arrays")
+}, "Float arrays");
+
+test(function() {
```

Comment:
Can you please update `test/fixtures/wpt/README.md` with the commit hash of this change?

Reply:
My bad, this change shouldn't be in this PR. I'll remove it.
This particular test has one small flaw: `new Float32Array(65537)` and `new Float64Array(65537)` are at least misleading here, since QuotaExceededError must be based on their `.byteLength`, not `.length`. I'm planning to submit a proper patch to wpt.